### ISSUE TITLE
Video ended completion not working when Dragon Web Extension installed

### ISSUE THREAD

**Issue by paulstevendev** [adaptlearning/adapt-vimeo#26]
A client has reported that Vimeo videos are not being recorded as complete on their PCs. All PCs have the dragon web extension installed for accessibility reasons and this is locked down so the user cannot disable this.

The videos in their courses all need to be watched to the end so the completion criterial is "ended"

This completion is not happening when it gets to the end of the video.

### Your environment
Adapt authoring tool 0.11.1.
Adapt framework v5.24.8 
Tested in latest Chrome and Edge
Windows 10 PC

### Steps to reproduce
Install the Dragon Web Extension in Chrome or Edge browser
Create a course with a vimeo video component and spoor
Set the video completion to ended
Upload to LMS and test course completion 

### Expected behaviour
The course should complete after watching the video to the end

### Actual behaviour
The course does not complete

**Comment by swashbuck** [comment]
@paulstevendev Is this issue also happening with the `adapt-contrib-media` component?

**Comment by paulstevendev** [comment]
> @paulstevendev Is this issue also happening with the `adapt-contrib-media` component?

It is happening with the YouTube video component. I will check the Media component and report back

**Comment by paulstevendev** [comment]
I  have tested the Media component playing the exact same Vimeo video and the completion was recorded correctly at the end of the video. So hopefully this will give a clue to helping fix the Vimeo and YouTube players so they behave the same?
